ui/tests/acceptance/oidc-provider-test.js

```diff
@@ -125,13 +125,14 @@ module('Acceptance | oidc provider', function (hooks) {
   setupApplicationTest(hooks);
 
   hooks.beforeEach(async function () {
+    this.timestamp = new Date().getTime();
```

Comment:
So it looks like places where we're using a date to create a unique name or id you opted to use `new Date().getTime()` consistently? 

I like this! Makes it easy to discern when the date is relevant or just being used to timestamp and/or name a test item

Reply:
It seems like this was already the existing pattern...but I appreciate it! 😂

Reply:
Exactly! Even in places where we only used it once in the test, hopefully by establishing this pattern we can keep things consolidated. Another option is to replace all of those with `uuidv4()` since that's a dependency now. I think both are valid directions, and then `timestamp` is less likely to be misused